src/storage.rs

```diff
@@ -38,26 +38,50 @@ const RAW_DATA_COLUMN_FAMILY_NAMES: [&str; 14] = [
 const META_DATA_COLUMN_FAMILY_NAMES: [&str; 1] = ["sources"];
 const TIMESTAMP_SIZE: usize = 8;
 
+pub struct DBOptions {
```

Comment:
[Rust API Guidelines](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case)에 따르면 `DbOptions`로 해야 합니다.

> In `UpperCamelCase`, acronyms and contractions of compound words count as one word: use `Uuid` rather than `UUID`, `Usize` rather than `USize` or `Stdin` rather than `StdIn`.